### ISSUE TITLE
Add more levels

### ISSUE THREAD

**Issue by SohamKukreti** [SohamKukreti/omilia#34]
Currently there are 8 levels for each language. We would want to expand the game further by adding more levels.

**Comment by dhanvithnayak** [comment]
Can I be assigned this issue? I'd like to work on the English version.

**Comment by SohamKukreti** [comment]
Sure!

**Comment by dhanvithnayak** [comment]
> Currently there are 15 levels for each language. We would want to expand the game further by adding more levels.

Each language currently has only eight levels, right? Or am I mistaken?

**Comment by SohamKukreti** [comment]
> > Currently there are 15 levels for each language. We would want to expand the game further by adding more levels.
> 
> Each language currently has only eight levels, right? Or am I mistaken?

Yeah, there are actually only eight levels. I accidentally wrote the number of language prompts that exist in the game which are 15. I'll correct it right now

**Comment by SohamKukreti** [comment]
Feel free to reach out with any other queries.

**Comment by dhanvithnayak** [comment]
How many levels should I preferably add? Also, should I add any more elements, or is it enough if I work with the existing ones?

**Comment by SohamKukreti** [comment]
It's enough if you work with existing elements. 4 more levels would be cool, although it's up to you to decide how many you wish to add. Any creativity would be highly appreciated.

**Comment by dhanvithnayak** [comment]
Here are the levels I designed:


![level_11](https://github.com/SohamKukreti/omilia/assets/85876638/6bbe8f8c-4b2c-4691-b998-86efdff8c55f)
![level_10](https://github.com/SohamKukreti/omilia/assets/85876638/5a5e9077-7231-4724-8a9d-c98ec5a2d3e4)
![level_5](https://github.com/SohamKukreti/omilia/assets/85876638/3db859b6-d36a-422a-8f55-9f3036dffb7c)
![level_9](https://github.com/SohamKukreti/omilia/assets/85876638/55f8e76d-e80a-47f6-bc9a-0f5b54466f58)

Is there anything that I should change?

**Comment by SohamKukreti** [comment]
Looks good! Create a pr so that I can test them out.
